htdocs/core/class/html.form.class.php

```diff
@@ -1055,8 +1055,12 @@ public function select_incoterms($selected = '', $location_incoterms = '', $page
 				}
 			}
 			$out .= '</select>';
-
-			$out .= '<input id="location_incoterms" class="maxwidth100onsmartphone nomargintop nomarginbottom" name="location_incoterms" value="'.$location_incoterms.'">';
+			
+			if ($conf->use_javascript_ajax && empty($disableautocomplete)) {
+		    	$out .= ajax_multiautocompleter('location_incoterms', '', DOL_URL_ROOT.'/core/ajax/locationincoterms.php')."\n";
```

Comment:
The PR mention a file locationincoterms, but the file is not included into the PR.

Reply:
uploaded. Getting slowly to understand Github.

Which file shall I edit to add sql table (namely c_incoterms)?